patches/apriltags.patch

```diff
@@ -43,7 +43,7 @@ diff -urN ../apriltag-2015-03-18/CMakeLists.txt ./CMakeLists.txt
 diff -urN ../apriltag-2015-03-18/manifest.xml ./manifest.xml
 --- ../apriltag-2015-03-18/manifest.xml	1970-01-01 01:00:00.000000000 +0100
 +++ ./manifest.xml	2015-04-30 17:20:35.639306306 +0200
-@@ -0,0 +1,12 @@
+@@ -0,0 +1,14 @@
 +<package>
```

Comment:
I just check the number of line below the differ header and is bigger than 12.
This patch header is wrong anyway.